templates/coderd.yaml

```
@@ -230,7 +230,7 @@ data:
   privateKey: {{ genPrivateKey "ecdsa" | b64enc }}
   {{- end }}
 {{- end }}
-{{ if eq (include "coder.useServiceNext" .) "true" }}
+{{- if and ( eq (include "coder.useServiceNext" .) "true"  ) (merge .Values dict | dig "ingress" "enable" false) }}
```

```suggestion
{{- if and ( eq (include "coder.useServiceNext" .) "true"  ) ( eq (merge .Values dict | dig "ingress" "enable" false) false ) }}
```

This actually resolves the ingress.enable to the proper value to skip the fancy LB.